infrastructure/src/main/java/org/corfudb/infrastructure/CompactorService.java

```diff
@@ -0,0 +1,161 @@
+package org.corfudb.infrastructure;
+
+import com.google.common.util.concurrent.ThreadFactoryBuilder;
+import lombok.NonNull;
+import lombok.Setter;
+import org.corfudb.runtime.CorfuCompactorManagement.CheckpointingStatus;
+import org.corfudb.runtime.CorfuCompactorManagement.CheckpointingStatus.StatusType;
+import org.corfudb.runtime.CorfuRuntime;
+import org.corfudb.runtime.CorfuStoreMetadata.TableName;
+import org.corfudb.runtime.DistributedCompactor;
+import org.corfudb.runtime.collections.CorfuStore;
+import org.corfudb.runtime.collections.TxnContext;
+import org.corfudb.runtime.view.Layout;
+import org.corfudb.util.concurrent.SingletonResource;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.time.Duration;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.Executors;
+import java.util.concurrent.ScheduledExecutorService;
+import java.util.concurrent.TimeUnit;
+
+import static org.corfudb.runtime.view.TableRegistry.CORFU_SYSTEM_NAMESPACE;
+
+/**
+ * Orchestrates distributed compaction
+ * <p>
+ * Created by Sundar Sridharan on 3/2/22.
+ */
+public class CompactorService implements ManagementService {
+
+    private long compactionTriggerFreqMs = TimeUnit.MINUTES.toMillis(8);
+
+    @Setter
+    private static Duration livenessTimeout = Duration.ofMillis(60000);
```

Comment:
Duration.ofMinutes(1) ?

Reply:
So for tests, it is convenient to use millis. If it's seconds/minutes, the tests timeout now and then.

Reply:
Sorry the convenience to use millis was before changing it to Duration. It was initially long and hence the response. Will make it to minutes here.